### PR TITLE
chore(select, menu-item, nav-disclosure): refactor focus (VIV-1519)

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: libs/components/src/lib/menu-item/ui.test.ts-snapshots/snapshots-menu-item-Desktop-Firefox-linux.png
Comment (RichardHelm) [inline libs/components/src/lib/menu-item/ui.test.ts-snapshots/snapshots-menu-item-Desktop-Firefox-linux.png:1]: What happened to this?

Comment (rachelbt) [reply libs/components/src/lib/menu-item/ui.test.ts-snapshots/snapshots-menu-item-Desktop-Firefox-linux.png:1]: I added transparent to the box-shadow so the focus now is like it should be and not 3px

Comment (RichardHelm) [reply libs/components/src/lib/menu-item/ui.test.ts-snapshots/snapshots-menu-item-Desktop-Firefox-linux.png:1]: But why does it look like this in Firefox?
image

File: libs/components/src/lib/menu-item/menu-item.scss [hunk 1]
```diff
@@ -16,12 +16,12 @@
...
+@use "../../../../shared/src/lib/sass/mixins/focus" as focus-variables;
 
 
-@supports selector(:focus-visible) {
-	:host(:focus-visible) {
-		outline: none;
-	}
+:host(:focus-visible) {
+	@include focus.focus;
 }
```
Comment (RichardHelm) [inline libs/components/src/lib/menu-item/menu-item.scss:25]: I think this is not right, outline: none turned into focus style

**Review by RichardHelm — CHANGES_REQUESTED** [review]

**Review by RichardHelm — APPROVED** [review]

**Pull request merged**